changelog.asc

```diff
@@ -2,6 +2,45 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241025)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Anki upstream version 24.06.3 included
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
+** Improved experience for devices with larger screens
+*** Better utilization of screen space
+*** Better mouse support
+*** Better keyboard shortcuts support
+*** Drag and drop support on some parts of the application
+** Better support for cards flags management
+** SHA256 & SHA1 application certificate fingerprints can be viewed in repository README
+** Update audio focus with support for Android 15
+** Reduced application size
+** Improved accessibility
+*** Better support for larger text size
+*** Added info text for more UI elements
+** Allow native deck selection in statistics screen
+** Study options screen shows bury counts as desktop
+** App shows info about WebView version requirements at start
+** Latest translations from our community of https://crowdin.com/project/ankidroid[translators] (thank you!)
+* Fixes
+** Fixed several issues related to keyboard handling
+** Fixed out of memory errors when displaying many tags 
+** Fixed crash for tags with unicode characters 
+** Fixed several issues with the interaction between note editor and template editor
+** Fixed learn ahead limit not being applied after setting change
+** Fixed several issues related to statistics page
+** Fixed silent crash on startup due to fatal error
+** Fixed several issues related to TTS replay
+** Fixed several issues related to portrait/landscape screen orientation
+** Fixed multiple UI inconsistencies
+** Fixed several issues to the timebox in review screen
+** Fixed several issues around the previewer
+** Many other smaller fixes
```

Comment:
I do not feel strongly about this but I try to include humor from time to time when possible
I think most apps and most programming is too corporate and impersonal and it is just no fun

I think users like fun

Feel free to be like "no, seriously dude, it's not even funny" and remove it. Or add your own humor :-)

```suggestion
** Many other smaller fixes (still reading this? Congratulations, you made it to the end!)
```

Reply:
Love it!